apidoc/Titanium/Android/Intent.yml

```diff
@@ -1,10 +1,109 @@
+    indicates to the Android OS that your application can handle certain data types or URIs.
+    For details on using Intent Filters, see the
+    [Android Intent Filters guide](http://docs.appcelerator.com/titanium/latest/#!/guide/Android_Intent_Filters).
+
+    Intents can be used to start an activity, start a service or start a broadcast.
+
+      * To create an intent to start an **Activity**, use the <Titanium.Android.createIntent> method.
+      * To create an intent to start a **Broadcast**, use the <Titanium.Android.createBroadcastIntent> method.
+      * To create an intent to start a **Service**, use the <Titanium.Android.createServiceIntent> method.
+
+    You can create either an implicit intent or an explicit intent.
+
+    An *explicit intent* specifies the application or application component to launch. To create an
+    explicit intent, specify the Intent's `className` and `packageName` properties to specify the
+    application component to launch or the `url` property to specify a JavaScript file to handle
+    the data.
+
+    An *implicit intent* does not specify a particular application.  Android will present the options
+    to the user of which applications to launch if a default application was not selected to handle
+    a particular data type or content URI. To create an implicit intent, do not specify the
+    `className`, `packageName` or `url` properties.
+
+    Note that the parameters to create a Service Intent are different than the ones used to
+    create an Activity or Broadcast.  The properties and methods listed below
+    are used for Activity and Broadcast Intents.  See the <Titanium.Android.createServiceIntent> method
+    for the parameters to create a Service Intent.
+
+    #### Action
+
+    The `action` property specifies the action you want the activity to perform, or in the case of
+    broadcasts, the action that just completed you want to report.
+
+    Titanium exposes some of the Android Intent actions as the `Titanium.Android.ACTION_*` constants.
+    Note that some of these actions are for system-level broadcasts that only Android can send.
+    If Titanium has not exposed a particular constant, you can pass the string value listed in the
+    Android API reference guide instead.
+
+    You can also define your own custom action names. Use a reverse domain scheme to name the
+    action to avoid potential conflicts, for example, `com.appcelerator.action.LINT`.
+    Custom actions are only useful to communicate between your applications and application
+    activities using intents.
+
+    #### Data
+
+    The [data](Titanium.Android.Intent.data) property specifies a content URI you want the activity to handle.
+
+    The [type](Titanium.Android.Intent.type) property specifies a MIME type the activity can handle.
+
+    For Broadcast Intents, do not use the `data` or `type` properties. Use extras to pass data.
```

Comment:
Maybe include a 'See 'Extras' below' here? I wasn't sure what this meant until I read further. 

[aside] It sucks that JSDuck can't link to sections on API pages. I feel there's a lot of good content in API pages that's kind of hard to access, and I'm less inclined to link to it if I have to say "the such and such section in <link>". Definitely a strike against JSDuck.